packages/client/hmi-client/src/components/workflow/ops/funman/tera-constraint-group-form.vue

```diff
@@ -153,6 +178,30 @@ const variableOptions = computed(() => {
 			return [];
 	}
 });
+
+function generateExpression() {
+	const { constraintType, interval, variables } = props.config;
+	let expression = '';
+	for (let i = 0; i < variables.length; i++) {
+		let expressionPart = variables[i];
+		if (constraintType === ConstraintType.Increasing || constraintType === ConstraintType.Decreasing) {
+			expressionPart = `d/dt ${variables[i]}`;
+		}
+		if (i === variables.length - 1) {
+			if (constraintType === ConstraintType.LessThan || constraintType === ConstraintType.Decreasing) {
+				expressionPart += interval?.closed_upper_bound ? `\\leq` : `<`;
+				expressionPart += constraintType === ConstraintType.Decreasing ? '0' : `${interval?.ub ?? 0}`;
+			} else {
+				expressionPart += interval?.closed_upper_bound ? `\\geq` : `>`;
```

Comment:
Should it be lower_bound here?

Reply:
Yeah I did this wrong actually even though there is no lower_bound